dll/win32/imm32/imm.c

```diff
@@ -61,6 +61,33 @@ SHAREDINFO g_SharedInfo = { NULL };
 BYTE g_bClientRegd = FALSE;
 HANDLE g_hImm32Heap = NULL;
 
+static PWND FASTCALL ValidateHwndNoErr(HWND hwnd)
```

Comment:
That function should be in user32.
https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/user32/misc/misc.c;hb=84f7bee18f5842c63b0ad4389af8b7f49e375738#l384

Reply:
I would expect this to be in some private header, that our components can be used.
(I have seen this symbol in multiple pdb files)

Reply:
This function is different in the point that it uses the handle table from `USER32`.

Reply:
`USER32` and `IMM32` are separate modules. And the handle table is one component in one process. So we have to use the shared info to access the handle table.